### DISCUSSION TITLE
Promise not completing on create(), process staying alive

### DISCUSSION THREAD

**Discussion by SirCameron** [prisma/prisma#3574] [Q&A] [answered]
Hi all,

I recently got started with Prisma and so far all is good and nice.
There is one thing, and I don't know if it's something I'm doing wrong.

If I run something like this:
```js
prisma.model
    .create({
      data: {
        ...some data
      },
    })
    .then((result) => {
      console.log(result)
    });
```

The process should print out the result and then exit, right?
but it doesn't. I tested with other promises to make sure and the process exits correctly.

Is there something I'm missing here?
Client and schema are generated correctly. Data saves correctly to the DB (mysql on Ubuntu)

versions: 
node v14.6.0
@prisma/client ^2.6.2
@prisma/cli ^2.3.0
this is a CLI app on Ubuntu.

**Comment by ryands17** [comment, marked as answer]
Hey @SirCameron 👋 
Your `@prisma/client` and `@prisma/cli` need to be installed locally in the project and be on the same version otherwise you will face a lot of unwanted issues. You can update to use `2.6.2` for both.

Also, if it's just a standalone Prisma `create` call, you could do something like this:

```ts
prisma.model
    .create({
      data: {
        ...some data
      },
    })
    .then((result) => {
      console.log(result)
    })
   .finally(async () => await prisma.$disconnect());
```

This will clear all DB connections kept by Prisma and will exit the the standalone script that you're running. You can read more about that [here](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/connection-management#disconnect)

**Reply by SirCameron** [reply]
Thank you @ryands17 !

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
